evidence/cli/utils.go

```diff
@@ -0,0 +1,14 @@
+package cli
+
+import "github.com/jfrog/jfrog-cli-core/v2/common/commands"
+
+type execCommandFunc func(command commands.Command) error
+
+func exec(command commands.Command) error {
+	return commands.Exec(command)
+}
+
+var subjectTypes = []string{
+	EvdRepoPath,
```

Comment:
The `EvdRepoPath` can be non-exported (the same as `releaseBundle`).
Also, the type name is quite confusing, is it `subjectRepoPath` or just `repoPath`?
(we do not specify a path to the evidence itself)

Reply:
done